### PR TITLE
Make migrationScript and downgradeScript optional

### PR DESCRIPTION
**Pull request by djmitche**
We have had two revisions now with no upgrade/downgrade scripts, so this seems a reasonable tweak.

### CODE REVIEW

File: libraries/postgres/src/Version.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const Method = require('./Method');
 const { loadSql } = require('./util');
 
 const objMap = (obj, fn) => Object.fromEntries(Object.entries(obj).map(fn));
+const ALLOWED_KEYS = ['version', 'migrationScript', 'downgradeScript', 'methods'];
```
Comment (imbstack) [inline libraries/postgres/src/Version.js:7]: argh, you always catch my issues before I get to r- you

**Comment by djmitche** [comment]
Oh, good, point - I could check that if migrationScript is set then downgradeScript must be set.

**Comment by helfi92** [comment]
> Oh, good, point - I could check that if migrationScript is set then downgradeScript must be set.

That's a really good idea!

File: libraries/postgres/src/Version.js (hunk 1 above)
Comment (djmitche) [reply libraries/postgres/src/Version.js:7]: Python's `set` takes an iterator as an arg to its constructor, so I always forget that the JS `Set` doesn't..

**Pull request merged**